db/schema.rb

```diff
@@ -83,13 +83,6 @@
     t.index ["user_id"], name: "index_organizers_on_user_id"
   end
 
-  create_table "participants", force: :cascade do |t|
-    t.integer "attendee_id"
-    t.string "attendee_type"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
```

Comment:
I don't understand why this table is removed

Reply:
So i face the same problem in my Issue now maybe the Table just isn' t used anymore.